zulipterminal/model.py

```diff
@@ -142,8 +149,11 @@ def append_message(self, response: Dict[str, Any]) -> None:
         response['flags'] = []
         if hasattr(self.controller, 'view') and self.update:
             self.index = index_messages([response], self, self.index)
-            msg_w = create_msg_box_list(self, [response['id']])[0]
-
+            msg_w_list = create_msg_box_list(self, [response['id']])
+            if msg_w_list == []:
```

Comment:
Use if `not msg_w_list` since [that's preferred](https://stackoverflow.com/a/45778282/1225068) over other ways of doing it.